src/docs/reverting/making-recovery-usb.md

```diff
@@ -0,0 +1,52 @@
+---
+prev: flashing-stock
+next: booting-recovery-usb
+---
+
+# Making a Recovery USB
+
+::: danger
+Making a Recovery USB will **wipe all data** on said drive. Back up anything if you need to.
+:::
+
+For you to reinstall chromeOS, you will need to make a Recovery USB.
+There are two methods depending on the Operating System Installed.
+## Chromebook Recovery Utility (Windows/ChromeOS/MacOS)
+1. Download the [Chromebook Recovery Utility](https://chrome.google.com/webstore/detail/chromebook-recovery-utili/pocpnlppkickgojjlmhdmidojbmbodfm) from the Chrome Web Store.
+
+2. Launch the app, then follow on-screen instructions. If you are unable to find your device, you can select the "Select a model from a list" option.
+
+3. Select the USB drive you want to use.
+
+4. Wait for it to complete.
+
+## Manually Flash Recovery Image (Windows/ChromeOS/MacOS/Linux)
+
+1. Go to [cros.tech](https://cros.tech/) and find your device.
+
+2. Select on "Recovery Images" and select the latest one.
+
+3. Extract the file.
+
+**Windows**
```

Comment:
Can we use `### Windows` (or `#### Windows`) for this section and linux and macOS/chromeos too

(macOS is spelled with a lowercase m)